recipe/meta.yaml

```diff
@@ -22,11 +21,12 @@ requirements:
   host:
     - pip
     - python
-    - setuptools >=45.0
-    - setuptools-scm >=6.0.1
+    - setuptools
+    - setuptools-scm 7.0.4
+    - wheel
   run:
     - python
-    - importlib_metadata
+    - importlib_metadata  # [py<38]
```

Comment:
```suggestion
```

I don't see it in https://codeberg.org/miurahr/pyppmd/src/tag/v1.0.0/pyproject.toml

Reply:
@cbouss that's an upstream mistake, it is not listed, but still used for python<3.8:
https://codeberg.org/miurahr/pyppmd/src/branch/main/src/pyppmd/__init__.py#L6

however, as we don't build for such old python versions, I think that we could remove. I kept it for consistency with other recipes.

Opinions?

Reply:
Ok, keeping it is a good call then.